grpc/proto/v1/batch_delete.proto

```diff
@@ -15,6 +15,7 @@ message BatchDeleteRequest {
   bool dry_run = 4;
   optional ConsistencyLevel consistency_level = 5;
   optional string tenant = 6;
+  int64 deletion_time_unix_milli = 7;
```

Comment:
How would this be useful? When an object is deleted I assume it is gone? Or can I somehow still get it?

Reply:
when objects are deleted, a tombstone is maintained and even preserved during compaction as `lsmkv.WithKeepTombstones(true)` is hard set during object store initialization.
Internally the method `WasDeleted` checks if keepTombstones is set.

Reply:
so you added it to the message to the batch delete request - what changes when I do a batch delete? And how would the user know what to put in there?

Reply:
fair point, just followed how PutObject was implemented but I realized that for the single object deletion the deletion time is not exposed so it can be done in the same manner for BatchDelete, thanks